### ISSUE TITLE
[a11y][FxA] Review a11y of non-standard FxA flows

### ISSUE THREAD

**Issue by mcomella** [mozilla-mobile/firefox-tv#2936]
Follow-up to #2773.

### Vision statement / What / Requirements
It's been a while since we've checked our ability to be used with a screen reader: we should ensure it's working. In particular, we just finished the Send Tab functionality so we ensure that is working too.

### Impact
Screen reader users may be unable to use these features without this testing.

### Acceptance criteria
*Other screens were tested in https://github.com/mozilla-mobile/firefox-tv/issues/2773*
- Test out non-standard FxA flows
  - Create account
  - Invalid email, password
  - Enter an email that doesn't have an account
  - Use "Forgot password" option
